File: IntroPC/Tarea 6/Rutina_perfectos/Fperfecto.cpp
#include <cstdio>
using namespace std;
int Fperfecto (int num)
{
  int div,sum=0;
  int i,cond;
  for (i=1;i<=(num-1);i++)
    {
      cond=num%i;
      if (cond == 0)
	{
	  sum+=i;
	}
    }
  if (sum==num)
    {
      return 1; 
    }
  else
    {
      return 0;
    }
}

File: IntroPC/Tarea 6/Rutina_perfectos/Fperfecto.py
#! /usr/bin/env python

def Fperfecto(num):
    i=1
    suma=0
    while i<=(num-1):
        cond=num%i
        if cond == 0:
            suma+=i
        i=i+1   				 
        
    if (suma==num):
        return 1 
        
    else:
        return 0
    
